app/v2/utils/StreamWithFile.scala

```diff
@@ -39,11 +41,21 @@ trait StreamWithFile {
     src: Source[ByteString, _]
```

Comment:
do we need size for withReusableSource() as well? if not better not to change this method

Reply:
Here we are not using size is you can see just returning source. This part is updated to minimize the code delicacy.